src/e2e-test/java/io/cdap/plugin/salesforcestreamingsource/stepsdesign/DesignTimeSteps.java

```diff
@@ -50,11 +73,59 @@ public void createNewLeadInSalesforce() throws UnsupportedEncodingException {
     lead.put("LastName", "LLname_" + uniqueId);
     lead.put("Company", uniqueId + ".com");
 
-    SalesforceClient.createLead(lead, "Lead");
+    SalesforceClient.createObject(lead, "Lead");
   }
 
   @Then("Enter unique Topic name as a Runtime argument value for key: {string}")
   public void fillUniqueTopicNameInRuntimeArguments(String runtimeArgumentKey) {
     SalesforcePropertiesPageActions.fillUniqueTopicNameInRuntimeArguments(runtimeArgumentKey);
   }
+
+  @Then("Update fields in salesforce")
+  public void updateFieldsInSalesforce() {
+    String uniqueRecordId = SalesforceClient.queryObjectId(SObjects.AUTOMATION_CUSTOM__C.value);
+    SalesforceClient.updateObject(uniqueRecordId, SObjects.AUTOMATION_CUSTOM__C.value);
+  }
+
+
+  @And("Create a new Custom Record in salesforce using REST API")
+  public void createANewCustomRecordInSalesforceUsingRESTAPI() throws UnsupportedEncodingException,
+    InterruptedException {
+    TimeUnit time = TimeUnit.SECONDS;
+    time.sleep(120);
```

Comment:
Why do we need a sleep in creating record ?

Reply:
We apply wait here because without wait it is not showing any record transferring in target table.